packages/design-system/src/components/Banner/style.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../scss/variables/colors';
+
+$base-class: 'banner';
+
+.#{$base-class} {
+  &--icon-close {
+    height: 24px;
+    width: 24px;
+    cursor: pointer;
+    position: absolute;
+    top: 17px;
+    right: 20px;
+  }
+
+  &--content--small {
```

Comment:
how to handle rwd?

Reply:
<img width="472" alt="Design System - LiveChat API Developers Docs and Guides 2020-05-27 12-46-17" src="https://user-images.githubusercontent.com/24607370/83010135-19c19e00-a018-11ea-9b47-44c370d1babc.png">

Reply:
The question was not answered.

Reply:
maybe we could add some notice about handling RWD? I know that it's hard to handle it in DS because this component will be inside a container always. Also, its size property is not related to the size of the window. Without using JS and some listeners it's hard to achieve it.